Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.Properties.cs

```diff
@@ -625,7 +625,7 @@ public string UriPrefix
         /// <summary>
         /// Holds a list of hyperlinks we are listening to.
         /// </summary>
-        private readonly List<Hyperlink> _listeningHyperlinks = new List<Hyperlink>();
+        private readonly List<object> _listeningHyperlinks = new List<object>();
```

Comment:
It would be nice to wrap this into a new class, or tuple or something, for Type safety.

Reply:
You could change it to UIElement (I think).

Reply:
Image is derived from UIElement. Pretty sure Hyperlink is not.

Reply:
Ah, that's right. I'm ok with this being how it is right now unless @WilliamABradley has any concerns